lib/logstash/outputs/opensearch/http_client/manticore_adapter.rb

```diff
@@ -157,6 +183,14 @@ def sign_aws_request(request_uri, path, method, params)
       params[:headers] =  params[:headers].merge(signed_key.headers)
     end
 
+    def params_basic_auth(params)
+      params[:auth] = {
+        :user => get_user(),
+        :password => get_password(),
+        :eager => true
+      }
```

Comment:
we should do something similar to https://github.com/opensearch-project/logstash-output-opensearch/pull/75/files#diff-4d45cd5f98df68872302f3ef7a840f3d6df9c28fa1ac8f2469c0b8bbbd0f9344L119